### ISSUE TITLE
Allow to use PollSelector on BSD as well

### ISSUE THREAD

**Issue by normanmaurer** [jnr/jnr-enxio#16]
PollSelector should work on linux and bsd. Could you allow to use PollSelector on BSD as well ? Maybe by adding another SelectoProvider which always return the PollSelector ?

**Comment by headius** [comment]
Indeed. Let's do this for 0.14. Help?

**Comment by Low-power** [comment]
I think this should be made user-configurable by reading a system property, so the user may switch to a non-default Selector implementation by adding a `-Djnr.enxio....` option to JVM.

**Comment by headius** [comment]
@Low-power Yes that seems reasonable. I think the only tricky bit is making sure the mappings also work on BSD.
